content/en/serverless/installation/go.md

```diff
@@ -30,17 +30,46 @@ go get github.com/DataDog/datadog-lambda-go
 1. Set environment variable `DD_FLUSH_TO_LOG` to `true`.
 1. Enable [AWS X-Ray active tracing][4] for your Lambda function.
```

Comment:
```suggestion
2. Enable [AWS X-Ray active tracing][4] for your Lambda function.
```

Reply:
Thanks for catching this, I believe the best practice is actually use `1` for all numbered bullet points for easy updates, but I'm going to fix it for consistency since we do use sequencing numbers in other places.